docs/source/rbac/tech-implementation.md

```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://auth0.com/docs/manage-users/access-control/rbac) allows for requesting tokens with specific permissions.
```

Comment:
This is not the right link for JupyterHub RBAC.

Reply:
ok Sir @minrk

Reply:
Sir @minrk i did the necessary changes as requested.